package.json

```diff
@@ -39,6 +39,7 @@
         "generateNonCodeFiles": "npm run generateNonCodeFiles -w packages/ --if-present"
     },
     "devDependencies": {
+        "@aws-toolkits/telemetry": "^1.0.239",
```

Comment:
I think telemetry is fine as a dev dependency, but was this intentional

Reply:
Yes, its only needed to generate definitions right? So only for building.